docs/src/pages/quasar-cli/quick-start.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Quick Start
+desc: Get up and developing a Quasar app in less than 3 minutes.
```

Comment:
```suggestion
desc: Get up and develop a Quasar app in less than 3 minutes.
```

```suggestion
desc: Get up and start developing a Quasar app in less than 3 minutes.
```

One of the suggestions above or maybe something else.

Reply:
The original is correct. 

Scott

Reply:
I think "Create a Quasar app..." Would read better here and it's less verbose.